### DISCUSSION TITLE
RingProgress display issue when root font size changes

### DISCUSSION THREAD

**Discussion by Trav1sRen** [mantinedev/mantine#6456] [Q&A]
If I set root font size according to the viewport size like
`<html lang="en" style={{ fontSize: 'clamp(14px, 0.95vw, 18px)' }}>`
The RingProgress label is not centered anymore and the ring svg exceeds the max width and height of the component root div when the `rem` size is less than `16px`

How could I make it display as normal?

**Comment by rtivital** [comment]
It is a known issue with the component that cannot be fixed. You can use different font-size in px on html element or use DonutChart instead.

**Reply by Trav1sRen** [reply]
What do u mean "use different font-size in px on html element"?

**Reply by rtivital** [reply]
`<html lang="en" style={{ fontSize: '16px' }}>`

**Reply by Trav1sRen** [reply]
Sir, u replied an empty message...

**Reply by rtivital** [reply]
`<html lang="en" style={{ fontSize: '16px' }}>`
